src/Home.tsx

```diff
@@ -12,51 +12,48 @@ import {
 import styles from './styles';
 
 import DaySelector from './components/DaySelector';
-import MealsComponent from './components/Meals';
+import Meals from './components/Meals';
 
-type DayOfWeek = {
-  short: string;
-  full: string;
-};
+export enum MealCategories {
+  BREAKFAST = 'Breakfast',
+  LUNCH = 'Lunch',
+  DINNER = 'Dinner',
+}
+
+export enum Day {
+  Sun = 'Sunday',
+  Mon = 'Monday',
+  Tue = 'Tuesday',
+  Wed = 'Wednesday',
+  Thu = 'Thursday',
+  Fri = 'Friday',
+}
 
 const HomePage: React.FC = () => {
-  const [meals, setMeals] = useState<string[]>(['', '', '']);
+  const [meals, setMeals] = useState<string[]>([]);
   const [errorMessage, setErrorMessage] = useState<string>('');
-  const [selectedDay, setSelectedDay] = useState<string | null>(null);
+  const [selectedDay, setSelectedDay] = useState<Day | null>(null);
 
-  const mealCategories: string[] = ['Breakfast', 'Lunch', 'Dinner'];
-  const daysOfWeek: DayOfWeek[] = [
-    { short: 'Sun', full: 'Sunday' },
-    { short: 'Mon', full: 'Monday' },
-    { short: 'Tue', full: 'Tuesday' },
-    { short: 'Wed', full: 'Wednesday' },
-    { short: 'Thu', full: 'Thursday' },
-    { short: 'Fri', full: 'Friday' },
+  const appButtons = [
```

Comment:
maybe "homeButtons"? or something of that sort

Reply:
ok